### PR TITLE
Adding Button Group Samples.

### LINKED ISSUE #135: Create Button Group user story and update current topic based on the article guidance.
Article guidance - https://github.com/IgniteUI/igniteui-docfx/blob/master/.github/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by npaunov**
Closes #135

### CODE REVIEW

File: en/components/buttonGroup.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+---
+title: Button Group Component
+_description: Provides button group functionality to developers and also allows horizontal/vertical alignment, single/multiple selection with toggling.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Button Group components, Angular Button Group controls, Angular Buttons components, Angular Buttons controls
+---
+
+## Button Group
+
+The **igx-ButtonGroup** component provides a button group functionality to developers and also allows horizontal/vertical alignment, single/multiple selection along with toggling. The igx-ButtounGroup component makes use of the [igxButton directive.](button.md)
```
Comment (ig-ealbert) [inline en/components/buttonGroup.md]: I still don't think that you can link to markdown files on production.  Markdown is compiled into HTML, right?

File: en/components/buttonGroup.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        ...
+        IgxButtonGroupModule,
+        ...
+    ]
+})
+export class AppModule {}
+```
+### Examples
+
+Use `igx-ButtonGroup` to organize buttons into an for Angular styled button group.
```
Comment (ig-ealbert) [inline en/components/buttonGroup.md]: into an?  no "for"?

File: en/components/buttonGroup.md (hunk 1 above)
Comment (tiliev) [reply en/components/buttonGroup.md]: @ig-ealbert Yes, .md is compiled into .html, the compiler will create an <a> tag out of this - "[igxButton directive.](button.md)". This is the correct way of linking between .md files.

File: en/components/buttonGroup.md (hunk 2 above)
Comment (npaunov) [reply en/components/buttonGroup.md]: This has bees addressed in #157

**Pull request merged**